modules/react-fiber/src/extend.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  MapView,
+  OrthographicView,
+  OrbitView,
+  FirstPersonView,
+  _GlobeView as GlobeView
+} from '@deck.gl/core';
+import {
+  S2Layer,
+  QuadkeyLayer,
+  TileLayer,
+  TripsLayer,
+  H3ClusterLayer,
+  H3HexagonLayer,
+  Tile3DLayer,
+  TerrainLayer,
+  MVTLayer,
+  GeohashLayer
+} from '@deck.gl/geo-layers';
+import {
+  ArcLayer,
+  BitmapLayer,
+  IconLayer,
+  LineLayer,
+  PointCloudLayer,
+  ScatterplotLayer,
+  ColumnLayer,
+  GridCellLayer,
+  PathLayer,
+  PolygonLayer,
+  GeoJsonLayer,
+  TextLayer,
+  SolidPolygonLayer
+} from '@deck.gl/layers';
+import type {Instance} from './types';
+
+// IDEA: we can technically move this purely to userland so that a user is defining precisely
```

Comment:
Articulating this more, as a developer I could in theory only `extend` explicitly what I am using in my app e.g. say I am only using a MapView and ScatterplotLayer:

```
extend({
  MapView,
  ScatterplotLayer
});
```

This would allow the bundle size to shrink (in theory) for the package consumer.